packages/reactive-element/src/reactive-element.ts

```diff
@@ -728,21 +729,22 @@ export abstract class ReactiveElement
     key: string | symbol,
     options: PropertyDeclaration
   ): PropertyDescriptor | undefined {
+    const {get, set} = getOwnPropertyDescriptor(this.prototype, name) ?? {
+      get(this: ReactiveElement) {
+        return this[key as keyof typeof this];
+      },
+      set(this: ReactiveElement, v: unknown) {
+        (this as unknown as Record<string | symbol, unknown>)[key] = v;
+      },
+    };
     return {
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      get(): any {
-        return (this as {[key: string]: unknown})[key as string];
+      get() {
+        return get!.call(this);
```

Comment:
Does `get` need the `!`? It looks like it should be definitely assigned. Same with `set` below.

Reply:
my understanding is that `f.call()` should be able the same speed as `f.bind()` and then a call. I think bind() used to actually be slower.